lib/std/math/big/int_test.zig

```diff
@@ -286,6 +286,30 @@ test "big.int string set bad base error" {
     try testing.expectError(error.InvalidBase, a.setString(45, "10"));
 }
 
+test "big.int string set empty string" {
+    var a = try Managed.init(testing.allocator);
+    defer a.deinit();
+
+    try a.setString(10, "");
+    try testing.expect(a.eqZero());
+}
+
+test "big.int string set underscores" {
+    var a = try Managed.init(testing.allocator);
+    defer a.deinit();
+
+    try a.setString(10, "__________");
+    try testing.expect(a.eqZero());
+}
+
+test "big.int string set base 36" {
+    var a = try Managed.init(testing.allocator);
+    defer a.deinit();
+
+    try a.setString(36, "efghijk");
```

Comment:
It would be nice to have at least one letter "z" as test case.

Reply:
done in the last commit